modules/express/README.md

```diff
@@ -77,14 +77,22 @@ Base URI: https://0.0.0.0:4000
 
 ### Building the docker container
 
-If you'd like to build the BitGo Express docker container yourself from the source code, first check out the latest release branch **rel/latest**, then run `docker build` from the project root. Here's the commands:
+If you'd like to build the BitGo Express docker container yourself from the source code, first check out the latest **master** branch, then run `docker build` from the project root. Here's the commands:
```

Comment:
Are we removing `rel/latest` as the core location for companies looking to build out the latest `stable` since `master` isn't a stable build?

Reply:
while that is true about the scope of this ticket, we still only want people only using `rel/latest` if they're going to build express themselves.